Index: test_steps.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import allure\r\nfrom selene.support.conditions import be\r\nfrom selene.support.shared.jquery_style import s\r\nfrom selene import browser, by\r\n\r\ndef test_github():\r\n    with allure.step('Открываем главную страницу'):\r\n        browser.open(\"/\")\r\n\r\n    with allure.step('Находим репозиторий'):\r\n        s('.header-search-button').click()\r\n        s('#query-builder-test').type(\"matygullinruslan/qa_guru_python_9.7\").press_enter()\r\n\r\n    with allure.step('Переходим по ссылке'):\r\n        s(by.link_text(\"matygullinruslan/qa_guru_python_9.7\")).click()\r\n\r\n        s('#issues-tab').click()\r\n        s(by.partial_text('#1')).should(be.visible)\r\n        ...\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/test_steps.py b/test_steps.py
--- a/test_steps.py	(revision b9781f4732d6722a8fc993ec8593749c147b5697)
+++ b/test_steps.py	(date 1705249050926)
@@ -11,10 +11,13 @@
         s('.header-search-button').click()
         s('#query-builder-test').type("matygullinruslan/qa_guru_python_9.7").press_enter()
 
-    with allure.step('Переходим по ссылке'):
+    with allure.step('Переходим по ссылке репозитория'):
         s(by.link_text("matygullinruslan/qa_guru_python_9.7")).click()
 
+    with allure.step('Открываем таб Issues'):
         s('#issues-tab').click()
+
+    with allure.step('Проверяем наличие Issues с номером 1'):
         s(by.partial_text('#1')).should(be.visible)
         ...
 
